server/build/babel/find-config-location.js

```diff
@@ -0,0 +1,16 @@
+import { join } from 'path'
+import buildConfigChain from 'babel-core/lib/transformation/file/options/build-config-chain'
```

Comment:
I admit as a Babel maintainer this makes me a little sad. If there's something we're not providing in the API, why not ask for it? Now if we change stuff like this we'll randomly break you with no way of knowing, and I'm 100% sure we'll get angry users coming to yell at us.